pyproject.toml

```diff
@@ -26,69 +26,88 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.8, <4.0"
 
-boto3 = "^1.24.11"
-botocore = "^1.27.11"
-pandas = "^1.2.0, <=1.5.1, !=1.5.0" # Exclusion per: https://github.com/aws/aws-sdk-pandas/issues/1678
-numpy = [
-    { version = ">=1.21.0 <=1.23.4", markers = "python_full_version >= '3.7.1' and python_full_version < '3.11.0'" },
-    { version = "^1.23.5", markers = "python_full_version >= '3.11.0'" },
-]
-pyarrow = ">=7.0.0, <10.1.0"
-redshift-connector = "~2.0.889"
-pymysql = "^1.0.0"
-pg8000 = "^1.29.0"
-openpyxl = "~3.0.0"
-requests-aws4auth = "^1.1.1"
-jsonpath-ng = "^1.5.3"
-progressbar2 = "^4.0.0"
-opensearch-py = "^2.0.0"
-gremlinpython = "^3.5.2"
-backoff = ">=2.0.0,<3.0.0"
```

Comment:
For some reason, `backoff` was added as a required dependency but used nowhere that I can see in the library. Removing has not caused the tests to fail either